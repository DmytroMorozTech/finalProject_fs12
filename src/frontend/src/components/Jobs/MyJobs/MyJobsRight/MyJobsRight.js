import ImgToJobsPage from '../../../../shared/ImgToJobsPage/ImgToJobsPage'
import styles from './styles'

function MyJobsRight () {
  const classes = styles()

  return (
    <div className={classes.mainRight}>
      <ImgToJobsPage/>
    </div>
  )
}

export default MyJobsRight
