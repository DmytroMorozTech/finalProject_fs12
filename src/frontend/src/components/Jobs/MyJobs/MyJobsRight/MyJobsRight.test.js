import { shallow } from 'enzyme'
import React from 'react'
import MyJobsRight from './MyJobsRight'

jest.mock('./styles', () => () => ({}))

const setUp = () => shallow(<MyJobsRight/>)

describe('should render MyJobsRight component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
