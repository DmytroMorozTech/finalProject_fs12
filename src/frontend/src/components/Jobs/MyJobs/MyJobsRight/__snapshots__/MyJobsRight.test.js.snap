// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MyJobsRight component to match snapshot 1`] = `
<div>
  <ImgToJobsPage />
</div>
`;
